src/styles/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $cell-width * 7 + $gap * 6;
```

Comment:
What are 7 and 6? Try to avoid magic numbers

Reply:
Done.